src/key_reset.c

```diff
@@ -710,8 +710,9 @@ PEP_STATUS receive_key_reset(PEP_SESSION session,
 
         // Basically, see if fpr is even in the database
         // for this user - we'll get PEP_ct_unknown if it isn't
-        if (ct_result == PEP_ct_unknown)
+        if (ct_result == PEP_ct_unknown) {
```

Comment:
YES!

Reply:
I personally prefer the inline version :^)
But if the main two ([leviathanch](https://github.com/leviathanch) & [planckdz](https://github.com/planckdz)) prefer to not use it, we should note it down somewhere along with the proper set up (iirc we did discuss it and end up on the standard being the Gnu one for grammar?).

Reply:
Yes, you are right of course.

* Keep pure formatting changes to a minimum outside the real code changes.
* Agree on a clang format file and call it a day. There is already one (`.clang-format`), please give feedback.

I can certainly live with the "braceless if" style, but it _will_ trip you up sooner or later.

Reply:
It's more a leftover from me debugging with printfs xD